src/sentry/integrations/gitlab/repository.py

```diff
@@ -78,3 +82,72 @@ def on_delete_repository(self, repo):
             if e.code == 404:
                 return
             installation.raise_error(e)
+
+    def compare_commits(self, repo, start_sha, end_sha):
```

Comment:
This is probably just me, but I interpret `compare_commits` a little differently – like, give me the discrete differences between two absolute "commits" (almost like a diff). But, I suppose you really can only do that by showing a list of all the changes (other commits) between them. Tricky.